tests/test_addon_detail.py

```diff
@@ -239,3 +239,65 @@ def test_extension_permissions(selenium, base_url, variables):
         assert permission.permission_description.is_displayed()
     addon.permissions.click_permissions_button()
     addon.wait_for_current_url('permission-request')
+
+
+@pytest.mark.nondestructive
+def test_more_info_card_header(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'More information' in addon.more_info.more_info_card_header
+
+
+@pytest.mark.parametrize(
+    'count, link',
+    enumerate(['Homepage', 'Support site', 'Support Email'])
+)
+@pytest.mark.nondestructive
+def test_more_info_support_links(selenium, base_url, variables, count, link):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert link in addon.more_info.addon_support_links[count].text
+    # excluding 'Support Email' since it doesn't open a new page when
+    # clicked; this is a mailto link which is not handled by the browser
+    if count != 2:
+        addon.more_info.addon_support_links[count].click()
+        # opens the homepage/support page provided by the developer
+        addon.wait_for_current_url('ghostery.com')
+
+
+@pytest.mark.nondestructive
+def test_more_info_version_number(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert addon.more_info.addon_version_number.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_more_info_addon_size(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert addon.more_info.addon_size.is_displayed()
```

Comment:
Can we check the size matches? Or maybe some part of the text matches?

Reply:
@jrbenny35 I've tried to do a file size comparison in https://github.com/mozilla/addons-release-tests/pull/201/commits/21fa92bfaf25241bc6e35a12eda6929c2e062074. This seems to work well. Let me know if you have other suggestions.